runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -429,7 +431,7 @@ public static CrypticClue forText(String text)
 				return clue;
 			}
 		}
-
+		log.warn("Clue found with unmatchable text: {}", text);
```

Comment:
Would this be just spam in log in case the clue is actually not cryptic clue but some other one?